src/main/scala/com/softwaremill/realworld/articles/ArticlesRepository.scala

```diff
@@ -0,0 +1,38 @@
+package com.softwaremill.realworld.articles
+
+import zio.ZLayer
+
+import java.time.ZonedDateTime
+
+
+class ArticlesRepository {
+
+  // TODO user proper db or create in-memory thread-safe store
+  private val articles: List[StoredArticle] = List(
+    StoredArticle(
+      1,
+      "dummy-article",
+      "Dummy Title",
+      "Represents a dummy article.",
+      "Dummy body",
+      List("dummy", "article"),
+      ZonedDateTime.parse("2016-02-18T03:22:56.637Z"),
+      ZonedDateTime.now(),
+      10),
+    StoredArticle(
+      2,
+      "dummy-article-2",
+      "Dummy Title 2",
+      "Represents a dummy article 2.",
+      "Dummy body 2",
+      List("dummy", "article", "2"),
+      ZonedDateTime.now(),
+      ZonedDateTime.now(),
+      10))
```

Comment:
nit: it is a stub but I would still advice on using the real data there, if possible, that are referred in the API documentation https://realworld-docs.netlify.app/docs/specs/backend-specs/endpoints/#list-articles

Reply:
Either way, I'm getting rid of the stubs in the next PRs.

Reply:
Discussed offline. We can close it now and go back to it when DB gets introduced.